src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Tracer } from '@aws-lambda-powertools/tracer';
+import {
+  BatchGetItemCommand,
+  BatchGetItemCommandInput,
+  BatchWriteItemCommand,
+  BatchWriteItemCommandInput,
+  DeleteItemCommand,
+  DeleteItemCommandInput,
+  DynamoDBClient,
+  DynamoDBClientConfig,
+  GetItemCommand,
+  GetItemCommandInput,
+  PutItemCommand,
+  PutItemCommandInput,
+} from '@aws-sdk/client-dynamodb';
+
+export class MhDynamoDbClient {
+  private readonly client: DynamoDBClient;
+
+  constructor(config: DynamoDBClientConfig) {
+    const tracer = new Tracer();
+    this.client = tracer.captureAWSv3Client(new DynamoDBClient(config));
```

Comment:
I think the constructor needs to take the `captureAWSv3Client` function as an option since we don't want to create a new tracer here, it should be part of this package.

Reply:
Should be some sort of options object where we can pass multiple options though?

Reply:
Good point. I will add it as a param. @jens-peterolsson you are right, there will be an options object, I just thought we'd align on the dynamo-related stuff first.

Reply:
I added my thoughts and some pointers 👍